types/react-native/index.d.ts

```diff
@@ -1667,26 +1667,31 @@ export interface TextInputProps extends ViewProps, TextInputIOSProps, TextInputA
  */
 interface TextInputState {
     /**
+     * @deprecated Use currentlyFocusedInput
      * Returns the ID of the currently focused text field, if one exists
      * If no text field is focused it returns null
      */
     currentlyFocusedField(): number;
 
     /**
-     * @deprecated Use ref.focus instead
-     * @param TextInputID id of the text field to focus
+     * Returns the ref of the currently focused text field, if one exists
+     * If no text field is focused it returns null
+     */
+    currentlyFocusedInput(): React.ElementRef<HostComponent<any>>;
```

Comment:
Upstream uses `mixed`, so this should be `unknown`.

```suggestion
    currentlyFocusedInput(): React.ElementRef<HostComponent<unknown>>;
```